PoshSSDTBuildDeploy/Functions/PublishDatabaseDeployment.ps1

```diff
@@ -14,46 +14,31 @@ Function Publish-DatabaseDeployment {
 
 
     )
-    if ($ScriptPath) {
-        if (-not (Test-Path $ScriptPath)) {
-            Throw "Script Path Invalid!"            
-        }
-    }
+    
     Write-Verbose 'Testing if DACfx was installed...'
-    Write-Verbose $dacfxPath
-    if (!$dacfxPath) {
```

Comment:
else statements arent required because if the if is true it will throw and exception and stop in anyway. So its safe and cleaner not to have the else. Also allows to move all these checks to be executed first